internal/core/items/text.rs

```diff
@@ -487,8 +487,13 @@ pub struct TextInput {
     pub letter_spacing: Property<LogicalLength>,
     pub width: Property<LogicalLength>,
     pub height: Property<LogicalLength>,
+    /// The following two properties are the actual cursor and anchor positions and they're kept in
+    /// bounds with the text by means of a binding installed. Never set them directly, but use the
+    /// helper functions instead.
     pub cursor_position_byte_offset: Property<i32>,
     pub anchor_position_byte_offset: Property<i32>,
+    pub unchecked_cursor_position_byte_offset: Property<i32>,
+    pub unchecked_anchor_position_byte_offset: Property<i32>,
```

Comment:
We could have a `#[repr(C)] struct CursorPosition { cursor_pos: i32, anchor: i32 }` ` 
then we can put that in a Property.
(But doesn't have to be in this PR, this is just a thought)